packages/devextreme/js/ui/drop_down_editor/ui.drop_down_editor.js

```diff
@@ -291,6 +291,16 @@ const DropDownEditor = TextBox.inherit({
         return 'none';
     },
 
+    _toggleAriaInvalid() {
+        const fieldData = this._fieldRenderData();
+
+        if(fieldData === null) {
+            this.setAria('invalid', 'true');
+        } else {
+            this.setAria('invalid', 'false');
```

Comment:
should we set aria-invalid even if empty value is allowed? i mean if it's not required